tests/test_automatic_fields.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test that when no fields are selected for a stream, automatic fields are still replicated
+"""
+from tap_tester import runner
+from base import MambuBaseTest
+
+class AutomaticFieldsTest(MambuBaseTest):
+    """
+    Test that when no fields are selected for a stream, automatic fields
+    are still replicated
+    """
+
+    @staticmethod
+    def name():
+        return "tap_tester_mambu_automatic_fields_test"
+
+    def untestable_streams(self):
+        return set([
+            "communications", # Need to set up Twilio or email server to send stuff
+        ])
+
+    def verify_is_selected(self, is_selected):
+        """
+        Override the default `verify_is_selected` because we only want automatic fields selected
+        """
+        self.assertFalse(is_selected)
```

Comment:
I think we can drop this since it is defined in base.py

Reply:
This is the one I need to use for this test